test/fixtures/builder-linode/minimal.json

```diff
@@ -0,0 +1,28 @@
+{
+    "variables": {
+      "linode_token": "{{env `LINODE_TOKEN`}}"
+    },
+    "builders": [
+      {
+        "type": "linode",
+        "linode_token": "{{user `linode_token`}}",
+
+        "region": "us-central",
+        "swap_size": 0,
```

Comment:
@displague I've got a pull request on your branch to bump this to `256`. The boot hangs for 1m30s while it waits for swap to be available.  Upping this to `256` avoids this, and there are already checks in place later to ignore swap when imagizing.